hugr-core/Cargo.toml

```diff
@@ -20,6 +20,7 @@ workspace = true
 extension_inference = []
 
 [dependencies]
+hugr-sexpr = { path = "../hugr-sexpr", version = "0.1.0", features = ["derive"] }
```

Comment:
Why is this included as a dependency for core? I don't see any callsites.

Reply:
I used to test the crate by generating and pretty printing s-expressions generated from HUGRs, but then decided to do that in a later PR. This is a remnant of that.